demos/common/python/openvino/model_zoo/model_api/adapters/openvino_adapter.py

```diff
@@ -0,0 +1,133 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import logging as log
+from collections import deque
+
+try:
+    from openvino.inference_engine import IECore, get_version
+    openvino_absent = False
+except ImportError:
+    openvino_absent = True
+
+import ngraph
```

Comment:
Since the `ngraph` is part of OV and the OV import is in `try..` block, then this module should be in that block too. Or it could throw an error without `OpenvinoAdapter` instantiation.

Reply:
Moved ngraph to this block